packages/h/hopscotch-map/xmake.lua

```diff
@@ -0,0 +1,28 @@
+package("hopscotch-map")
+
+    set_homepage("https://github.com/Tessil/hopscotch-map")
+    set_description("A C++ implementation of a fast hash map and hash set using hopscotch hashing")
+
+    set_urls("https://github.com/Tessil/hopscotch-map/archive/v$(version).zip",
+             "https://github.com/Tessil/hopscotch-map.git")
+
+    add_versions("2.3.0", "56ce4ff67215656065ee1a08948533baf9447c4440196ea5133c024856006938")
```

Comment:
I saw git tag is v2.3.0. we can use `add_versions("v2.3.0")` and `/$(version).zip`. 

so we can also select tag version from git url automatically.